azure/durable_functions/orchestrator.py

```diff
@@ -100,13 +100,22 @@ def handle(self, context: DurableOrchestrationContext):
                     actions=self.durable_context.actions,
                     custom_status=self.durable_context.custom_status)
             except Exception as e:
+                exception_str = str(e)
                 orchestration_state = OrchestratorState(
                     is_done=False,
                     output=None,  # Should have no output, after generation range
                     actions=self.durable_context.actions,
-                    error=str(e),
+                    error=exception_str,
                     custom_status=self.durable_context.custom_status)
 
+                # Create formatted error, using out-of-proc error schema
```

Comment:
I feel like this should be captured in it's own PR, as it is a rather major change.

Reply:
This is actually a changed that's already merged, here: https://github.com/Azure/azure-functions-durable-python/pull/196

Perhaps the GitHub UI is acting up, in part because this PR is a bit old by now :)

Reply:
Weird...

I definitely see it in dev, so disregard this.